syft/format/internal/cyclonedxutil/helpers/component.go

```diff
@@ -58,6 +59,16 @@ func EncodeComponent(p pkg.Package) cyclonedx.Component {
 	}
 }
 
+// TODO: we eventually want to update this so that we can read "supplier" from different syft metadata
+func encodeSupplier(_ pkg.Package, sbomSupplier string) *cyclonedx.OrganizationalEntity {
```

Comment:
the equivalent decoder support for this field I think is missing (getting supplier specified in a syft doc when decoding a cyclonedx one)

Reply:
might be worth updating the encoder-decoder cycle integration test to have a supplier specified in the source.

Reply:
Oh I missed this comment on my last pass. Let me get this added now.